media-api/app/lib/elasticsearch/filters.scala

```diff
@@ -22,6 +22,25 @@ object filters {
 
   def boolTerm(field: String, value: Boolean): TermQuery = termQuery(field, value)
 
+  /**
+   * Range query based on dates
+   * Note that `from` is _inclusive_ in this function
+   * @param field Field name to query
+   * @param from Lower bound for date (inclusive)
+   * @param to Upper bound for date (exclusive)
+   * @return Suitable query
+   */
+  def date(field: String, from: DateTime, to: DateTime): Query =
+    rangeQuery(field).gte(printDateTime(from)).lt(printDateTime(to))
+
+  /**
+   * Range query based on dates - handles optional to and from
+   * Note that `from` is _exclusive_ in this function
+   * @param field Field name to query
+   * @param from Lower bound for date (exclusive)
+   * @param to Upper bound for date (exclusive)
+   * @return Suitable query if at least one of `from` and `to` is defined; otherwise nothing
+   */
```

Comment:
not a fan that I've made the two overloads of `date` have different semantics on the `for` attribute :( 
how would feel about making the old/optional/bottom one also use `gte` (ie. an inclusive lower bound) ? I don't _think_ this should have any major impact - its usages are either for inexact fields (date taken, upload time etc.) or are used when filtering based on `now`

Reply:
based on your assessment of impact, perhaps go for that.
an alternative, with no risk would be to change your new function to be exclusive on the `from` too, then increment the day at the call site instead.